<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Ideas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="numGensLabel">
       <property name="text">
        <string>0 generations produced</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="selectDesign">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Full Factorial Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Taguchi Optimization Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plackett Burman Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Box Behnken Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latin Hyper Cube Design</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Central Composite Design</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="more">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Parameter settings</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelView">
     <property name="text">
      <string>Click arrows or table row to view Gen:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="previousGen">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>View previous</string>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="selectGenBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>Gen </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxKeep">
       <property name="toolTip">
        <string>Reduces the time for reloading</string>
       </property>
       <property name="text">
        <string>keep open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextGen">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>View next</string>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="parametersTable">
     <property name="toolTip">
      <string>Select view</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteGensButton">
     <property name="text">
      <string>Delete all generations</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
